src/main/java/seedu/address/storage/JsonAdaptedLog.java

```diff
@@ -36,7 +36,7 @@ public JsonAdaptedLog(@JsonProperty("entry") String entry,
     public JsonAdaptedLog(Log source) {
         entry = source.getEntry();
```

Comment:
changed based on the new Log.getAppointmentDate() implementation which return a AppointmentDate